src/script/view_model/CallingViewModel.ts

```diff
@@ -150,6 +157,44 @@ export class CallingViewModel {
       });
     };
 
+    const joinOngoingCall = async (call: Call) => {
```

Comment:
Was also considering this kind of api:
```ts
const joinOngoingCall = async (call: Call, onJoin?: (call: Call) => void) => {
  //do subconversation related stuff and update epoch info (only for mls conference)
  
  onJoin?.(call)
}
```

We could then call this function like: `joinOngoingCall(call, this.callingRepository.answerCall)`, not sure how important the order is, because in case of incoming call, the spec says we should call `wcall_incoming` first before joining subconversation.